backend/pages/utils/data_processing.py

```
@@ -4,7 +4,7 @@
 from .chart_classification import COLUMNS_TO_REMOVE, ROWS_TO_REMOVE
 import inflection
 
-def convert_df_to_json(df):
+def convert_df_to_json(df, precision):
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)